ajax/libs/js-popup/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "js-popup",
+  "filename": "js-popup.min.js",
+  "version": "1.0.0",
+  "description": "Simple popup messages for JavaScript/ReactJS.",
+  "homepage": "https://github.com/exelban/js-popup/",
+  "keywords": [
+    "react",
+    "reactjs",
+    "popup",
+    "popup-window",
+    "popup-messages",
+    "popup-windows",
+    "popup-messaging",
+    "window",
+    "window-popup",
+    "popups",
+    "js-popup",
+    "component",
+    "JavaScript",
+    "box/popup window",
+    "box window",
+    "dialog window",
+    "dialog popup",
+    "dialog popup window"
+  ],
+  "author": {
+    "name": "Serhiy Mytrovtsiy",
+    "email": "mitrovtsiy@ukr.net"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/exelban/js-popup.git"
+  },
+  "npmName": "js-popup",
+  "npmFileMap": [
+    {
+      "basePath": "release",
```

Comment:
sorry about that.
`npmFileMap` is a list of files to take from the npm tarball and host on cdnjs.
If use npm auto-update, it should modify as the following:
 `"basePath": "release",` --> `"basePath": "lib",`